api-reference/beta/resources/directoryrole.md

```diff
@@ -11,22 +11,22 @@ By default, directory roles are scoped to be tenant-wide.  However, directory ro
 |[Get directoryRole](../api/directoryrole_get.md) | [directoryRole](directoryrole.md) |Read properties and relationships of directoryRole object.|
 |[Add member](../api/directoryrole_post_members.md) |[directoryObject](directoryobject.md)| Add a user to the directory role by posting to the members navigation property.|
 |[List members](../api/directoryrole_list_members.md) |[directoryObject](directoryobject.md) collection| Get the users that are members of the directory role from the members navigation property.|
-|[List scoped administrators](../api/directoryrole_list_scopedadministrators.md) |[scopedRoleMembership](scopedrolemembership.md) collection| List the members of this directory role that are scoped to [administrative units](administrativeunit.md), through the scopedRoleMembership object collection.|
-<!--|[Remove member](../api/directoryrole_delete_members.md) |[directoryObject](directoryobject.md)| Remove a user from the directory role by posting to the members navigation property.|-->
+|[Remove member](../api/directoryrole_delete_member.md) |[directoryObject](directoryobject.md)| Remove a user from the directory role.|
```

Comment:
My bad.  Should change to "Remove a member...".  In the future this should support more than just users.